app/src/main/kotlin/com/wire/android/mapper/MessageMapper.kt

```diff
@@ -120,7 +120,7 @@ class MessageMapper @Inject constructor(
                     header = provideMessageHeader(sender, message),
                 )
             null -> null
-            UIMessageContent.PreviewAssetMessage -> null // Preview images messages should not be displayed
+            UIMessageContent.IncompleteAssetMessage -> null // Incomplete images messages should not be displayed
```

Comment:
I honestly am confused now what IncompleteASset means ? maybe adding a explanation to this type would be helpfull

Reply:
or maybe some name that could help determine what Incomplete means

Reply:
Ok, I added some descriptions